obspy/core/inventory/util.py

```diff
@@ -864,7 +863,7 @@ def _seed_id_keyfunction(x):
     return x
 
 
-def plot_inventory_epochs(plot_dict, outfile=None, colorspace=None, show=True,
+def plot_inventory_epochs(plot_dict, outfile=None, colorspace=None, show=False,
```

Comment:
I think usually we have `show=True` in all plots? I.e. having interactive plots pop up on default..